tests/serializers/test_natural.py

```diff
@@ -69,6 +69,27 @@ def natural_key_test(format, self):
     self.assertIsNone(books[1].object.pk)
 
 
+def natural_primary_keys_multi_table_test(format, self):
+    sub_class_instance_one = {'code': '1'}
+    sub_class_instance_two = {'code': '2'}
+
+    SubClass.objects.create(**sub_class_instance_one)
+    SubClass.objects.create(**sub_class_instance_two)
+
+    all_objects = list(SuperClass.objects.all()) + list(SubClass.objects.all())
+
+    string_data = serializers.serialize(
+        format, all_objects, use_natural_foreign_keys=True, use_natural_primary_keys=True,
+    )
+
+    for obj in serializers.deserialize(format, string_data):
+        if type(obj) == SubClass:
```

Comment:
I'd like if you could rework the assertion so it always runs. Currently it looks a bit brittle in that if the if condition never evaluates to true, the assertion won't be executed and the test will still pass.